app/src/store/cart.ts

```diff
@@ -178,6 +178,8 @@ export default function useCartStore() {
    */
   function addToCart(grantId: number | undefined) {
     if (grantId === undefined) return;
+    const idList = approvedGrantsPk.value as Array<number>;
+    if (!idList.includes(grantId)) return;
```

Comment:
If we're hiding the add to cart button for non-whitelisted grants, why do we need this? Seems like if executed this would result in silent failures that confuse the user (i.e. nothing happens when they try to add a grant to the cart)

Reply:
don't really need it but thought it could be used as an extra safety check idk